package.json

```diff
@@ -1,20 +1,13 @@
 {
   "dependencies": {
-    "@google-web-components/google-chart": "^5.0.2",
-    "@material/mwc-checkbox": "^0.27.0",
-    "@material/mwc-formfield": "^0.27.0",
-    "@material/mwc-icon": "^0.27.0",
-    "@material/mwc-icon-button": "^0.27.0",
-    "@material/mwc-list": "^0.27.0",
-    "@material/mwc-select": "^0.27.0",
-    "@material/mwc-textfield": "^0.27.0",
-    "@mdi/js": "^7.0.96",
-    "lit": "^2.3.1",
-    "superstruct": "^0.16.0",
+    "memoize-one": "^6.0.0",
+    "moment": "^2.29.4",
```

Comment:
We should avoid using moment. It's not meant for websites as it's a big dependency (at least last time I worked with it). Can you use `date-fns` instead ?

Reply:
Its not used by the website, its used by the generator.
But sure, can look at it

Reply:
ah looks like it's only used as part of eleventy -> should we move it to devDependencies ?